3ds/SDL/SDL-1.2.15.patch

```diff
@@ -759,13 +759,31 @@ index 000000000..bcb296d14
 +		SDL_PrivateJoystickButton (joystick, 7, SDL_PRESSED);
 +	}
 +	if ((key_press & KEY_START)) {
-+		SDL_PrivateJoystickButton (joystick, 8, SDL_PRESSED);
```

Comment:
This is why the Start button didn't work… I'm surprised nobody noticed it until now 😂